tests/test_dive_numbers.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+import glidertools.utils as gt_util
+
+from glidertools.load import seaglider_basestation_netCDFs
+
+
+# load some data
+filenames = "./tests/data/p542*.nc"
+
+names = ["ctd_depth", "ctd_time"]
+ds_dict = seaglider_basestation_netCDFs(filenames, names, keep_global_attrs=False)
+
+dat = ds_dict["sg_data_point"]
+depth = dat["ctd_depth"]
+time = dat["ctd_time"]
+
+
```

Comment:
I would suggest adding a parametrization here:

```python
@pytest.mark.parametrize('dive_depth_threshold, expected_max', [15, 599.5], [200, <expected value from old code>])
def test_find_correct_number_dives(dive_depth_threshold, expected_max):
    ...
    assert dives.max() == expected_max
```

That way we can assert that both the new and old behavior work.

Reply:
but the old behaviour incorrectly identified dives.

Reply:
Ah I see. Then please disregard.